python/core/additions/qgsfunction.py

```diff
@@ -51,18 +51,20 @@ def myfunc(values, *args):
     :param arg_count:
     :param group:
     :param usesgeometry:
+    :param handlesnull:
     :return:
     """
 
     class QgsPyExpressionFunction(QgsExpressionFunction):
 
         def __init__(self, func, name, args, group, helptext='', usesGeometry=True,
-                     referencedColumns=QgsFeatureRequest.ALL_ATTRIBUTES, expandargs=False):
+                     referencedColumns=QgsFeatureRequest.ALL_ATTRIBUTES, expandargs=False, handlesNull=False):
```

Comment:
Should we add some docs/unit tests here?

Reply:
Unit Tests: Sure. Docs, I'm not sure if that exists in an accessible way.

Reply:
Not sure if it is currently either - I'm just going off https://github.com/qgis/pyqgis/issues/11 that there's potentially future PyQGIS doc improvements which will expose classes like this too.

Reply:
As you might have noticed, there's also a commit for pyqgis docs in here. I'm hoping for the same.